src/visualizers/VisualizerHistogram.ts

```diff
@@ -0,0 +1,203 @@
+import {VisualizerExportModule} from '@/visualizers/VisualizerInterface'
+import {VisualizerDefault} from './VisualizerDefault'
+import {natlog} from 'c:/Users/devli/Desktop/Numberscope/frontscope/src/shared/math'
```

Comment:
This isn't going to work for other people on other computers.

Reply:
@Decos14, can you fix this import so that it's a relative import? I think it would be `../shared/math`.

Reply:
On my computer:

<img width="1110" alt="error" src="https://user-images.githubusercontent.com/60749003/199706126-ef2cd471-8b23-45e5-9e4c-924f6fc265d6.png">